core/common/src/rpc/IModelReadRpcInterface.ts

```diff
@@ -76,6 +76,7 @@ export abstract class IModelReadRpcInterface extends RpcInterface {
   /** @deprecated */
   public async getViewThumbnail(_iModelToken: IModelRpcProps, _viewId: string): Promise<Uint8Array> { return this.forward(arguments); }
   public async getDefaultViewId(_iModelToken: IModelRpcProps): Promise<Id64String> { return this.forward(arguments); }
+  public async getDefaultViewStateData(_iModelToken: IModelRpcProps, _modelIds?: CompressedId64Set): Promise<SerializedViewStateProps> { return this.forward(arguments); }
```

Comment:
Replace `modelIds` with an "options" object for future extensibility.

Reply:
Should those options propagate down to the viewcreatorhelper.getDefaultviewStateData method as well? I.e. replace modelIds with an "options" object ?

Reply:
I would expect so.